atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -116,8 +116,12 @@ def download_sources(self, urls, insecure=False, download_dir=SRPMS_DOWNLOAD_DIR
 
         req_session = get_retrying_requests_session()
         for url in urls:
-            download_url(url, dest_dir, insecure=insecure,
-                         session=req_session)
+            if isinstance(url, dict):
```

Comment:
Please keep the same structure for both codepaths, i.e. always dict

Reply:
ok